src/ForeElementMixin.js

```diff
@@ -135,7 +137,7 @@ export const foreElementMixin = superclass =>
         if (nodeType) {
           this.nodeset = evaluateXPathToFirstNode(this.ref, inscopeContext, this);
         } else {
-          this.nodeset = evaluateXPath(this.ref, inscopeContext, this);
+          [this.nodeset] = evaluateXPath(this.ref, inscopeContext, this);
```

Comment:
so the return of evaluateXPath will always be an array now - do i get this right?